.github/workflows/battery_plus.yaml

```diff
@@ -27,6 +27,27 @@ jobs:
       - name: "Build Example"
         run: ./.github/workflows/scripts/build-examples.sh android ./lib/main.dart
 
+  android_integration_test:
+    runs-on: ubuntu-latest
```

Comment:
Should be macos-latest (because it works better for emulators)

Reply:
Yes, I woke up and looked at the error message again. The error message reported suggested running on the macOS platform (as seen in my git commit message).

I was doing it for a test. After all, I didn't understand why you wrote it that way at the time.